docs/developer_guide.md

```diff
@@ -45,6 +45,15 @@ website and mixer changes.
   ./run_test.sh --setup_python
   ```
 
+- Python 3.12
```

Comment:
This section header seems at odds with the above `Python 3.11` section that reads: "Confirm the Python3 version is 3.11.x. Otherwise install/upgrade your Python and confirm the version". 

Can you make it more cohesive?

Reply:
So I think the python 3.11 guidance should just be that we need you to have AT LEAST python 3.11.
But if you have 3.12, then you also need to do this. 
I changed the wording to reflect that but let me know if you prefer to change it to just tell them to use 3.12 and always follow that additional setup.

Reply:
I agree using Python 3.11+ sounds to me!